Validator/Constraints/Unique.php

```diff
@@ -16,6 +16,35 @@
 #[Attribute(Attribute::TARGET_CLASS | Attribute::IS_REPEATABLE)]
 class Unique extends UniqueEntity
 {
-    /** @var string */
-    public $service = 'doctrine_odm.mongodb.unique';
+    /**
+     * @param array|string      $fields     The combination of fields that must contain unique values or a set of options
+     * @param bool|array|string $ignoreNull The combination of fields that ignore null values
+     */
+    public function __construct(
+        $fields,
+        string $message = null,
+        string $service = 'doctrine_odm.mongodb.unique',
+        string $em = null,
+        string $entityClass = null,
+        string $repositoryMethod = null,
+        string $errorPath = null,
+        bool|string|array $ignoreNull = null,
```

Comment:
```suggestion
     */
    public function __construct(
        $fields,
        string $message = null,
        string $service = 'doctrine_odm.mongodb.unique',
        string $em = null,
        string $entityClass = null,
        string $repositoryMethod = null,
        string $errorPath = null,
        bool $ignoreNull = null,
```
Now I see this, union types are introduced in PHP >= 8.0, so I guess we should stick to Symfony 5.4 constructor: https://github.com/symfony/symfony/blob/8492f10d29e929a83ccf6c306d40dbad69c9c80b/src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntity.php#L55